pkg/admission/clusterworkspacetypeexists/admission.go

```diff
@@ -99,6 +101,29 @@ func (o *clusterWorkspaceTypeExists) Admit(ctx context.Context, a admission.Attr
 	}
 
 	if a.GetOperation() == admission.Create {
+		// ensure the user's UID is recorded in annotations
+		if user, ok := genericapirequest.UserFrom(ctx); ok {
```

Comment:
is this different from `a.GetUser()` ?

Reply:
I did not know about that - let me use it instead